### PR TITLE
feat(ifr): new small Mistral model on catalog

### PR DESCRIPTION
**Pull request by tgenaitay**
### Your checklist for this pull request

- [X] Name your pull request according to the [contribution guidelines](https://github.com/scaleway/docs-content/blob/main/docs/CONTRIBUTING.md).

### Description

- Removed Llama2 models not consumed nor in prod anymore
- Added Mistral-7b doc
- Edited navigation

### CODE REVIEW

**Review by bene2k1 — APPROVED** [review]

**Review by ldecarvalho-doc — APPROVED** [review]

**Comment by bene2k1** [comment]
Wait for go by @tgenaitay before merge

**Comment by tgenaitay** [comment]
Let's go :D

**Pull request merged**
